spring-cloud/pom.xml

```diff
@@ -35,9 +35,10 @@
         <module>spring-cloud-archaius</module>
         <module>spring-cloud-functions</module>        
         <module>spring-cloud-vault</module>
-		<!-- <module>spring-cloud-security</module> --> <!-- Fixing in BAEL-10887 -->
-		<module>spring-cloud-task</module>
+        <!-- <module>spring-cloud-security</module> --> <!-- Fixing in BAEL-10887 -->
+        <module>spring-cloud-task</module>
         <module>spring-cloud-zuul</module>
+        <module>spring-cloud-zuul-fallback</module>
```

Comment:
instead of creating a new module, please check first if we can put everything inside spring-cloud-zuul, and isolate with a java package.

Reply:
Hi @rpvilao, as discussed and agreed over JIRA, a new module is required. However, I have updated the ports of the application as suggested by Josh.